torch/utils/model_dump/code.js

```diff
@@ -439,6 +439,114 @@ class OnePickleSection extends Component {
   }
 }
 
+function computeTensorMemory(numel, dtype) {
+  const sizes = {
+    "Byte": 1,
+    "Char": 1,
+    "Short": 2,
+    "Int": 4,
+    "Long": 8,
+    "Half": 2,
+    "Float": 4,
+    "Double": 8,
+    "ComplexHalf": 4,
+    "ComplexFloat": 8,
+    "ComplexDouble": 16,
+    "Bool": 1,
+    "QInt8": 1,
+    "QUInt8": 1,
+    "QInt32": 4,
+    "BFloat16": 2,
+  };
+  let dtsize = sizes[dtype];
+  if (!dtsize) {
+    throw new Error("Unrecognized dtype: " + dtype);
+  }
+  return numel * dtsize;
```

Comment:
This doesn't seem right (or I'm missing something), since numel * element size may not hold true for one of the bundled input tensors for example.

Reply:
This is the numel of the storage, not the tensor.  Of course, if we inflate the bundled input, that will allocate a new, larger tensor, but there's not much we can do about that here.